Device.go

```diff
@@ -258,6 +259,10 @@ func (d *device) GetPropertyManaged() (bool, error) {
 	return d.getBoolProperty(DevicePropertyManaged)
 }
 
+func (d *device) SetPropertyManaged(managed bool) error {
+	return d.setProperty(DevicePropertyManaged, dbus.MakeVariant(managed))
```

Comment:
setProperty already makes a dbus Variant out of the property value : https://github.com/Wifx/gonetworkmanager/commit/90c97fb09842e36dcb3c0c19709a33055b33c228#diff-7d1c2a3334601b6c1958aae0a594cba8R62

Reply:
Thanks for the review. I've updated my commit.